### PR TITLE
[@types/sdp-transform] Add a missing field from 'origin'

### PR DESCRIPTION
**Pull request by antonshwarts**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/clux/sdp-transform/blob/32da4469b85d1198c0524e3118827dd6085adc97/lib/grammar.js#L11>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [X] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## sdp-transform ([<kbd>unpkg</kbd>](https://unpkg.com/browse/sdp-transform@latest/))
was missing the following properties:
1. parseFmtpConfig
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1972180ef5a26c40e0272f9dd284833be8397909
</p>

**Comment by elibarzilay** [comment]
@loc @muenchow: ping?
@antonshwarts: what's the status of this?

File: types/sdp-transform/sdp-transform-tests.ts [hunk 1]
```diff
@@ -4,15 +4,31 @@ import {
...
+    session.media[0].fmtp[0].config = 'maxplaybackrate=48000;stereo=1;useinbandfec=1';
+    session.media[0].rtcpFb = [{ payload: 96, type: 'ccm', subtype: 'fir' }, { payload: 98, type: 'nack', subtype: 'rpsi' }];
+    session.media[0].rtcpFbTrrInt = [{ payload: 96, value: 100 }, { payload: 98, value: 100 }];
+    const sdp: string = write(session);
+}
+
+function test_origin_fields() {
+    const session: SessionDescription = parse('');
+    session.version = 0;
+    session.origin = {
```
Comment (loc) [inline types/sdp-transform/sdp-transform-tests.ts:25]: can you not just add this to the code above? why does it need to be separate?

**Comment by loc** [comment]
Hmm. Would've preferred my question getting answered before this merged, but I suppose I could've blocked it.

**Pull request merged**